src/event/HandlerList.php

```diff
@@ -130,26 +129,24 @@ public function getListenerList() : array{
 		}
 
 		$listenersByPriority = [];
+		$concurrentAsyncListenersByPriority = [];
```

Comment:
Are you sure we want 34-character-long variable names?

Reply:
The other option is to make `$listenersByPrioritybyType[0]`, `$listenersByPrioritybyType[1]`, and `$listenersByPrioritybyType[0]` but if we don't hardcode these values it would make it longer than before.

Reply:
Do we actually need to mention all four keywords ("concurrent", "async", "priority", "listener") in the same variable name?

Reply:
I don't think so, we have to find shorter terms.